test/solidity/Periphery/ServiceFeeCollector.t.sol

```diff
@@ -50,7 +50,11 @@ contract ServiceFeeCollectorTest is DSTest {
         uint256 fee = 0.015 ether;
 
         // Act
-        feeCollector.collectNativeGasFees{ value: fee }(137, address(0xb33f));
+        feeCollector.collectNativeGasFees{ value: fee }(
```

Comment:
I think it could be nice to have a test case that tests the behaviour we just had issues with to ensure that things work as intended